src/benchmark/tpch_benchmark.cpp

```diff
@@ -134,7 +137,8 @@ class TpchBenchmark final {
         _max_duration(max_duration),
         _output_file_path(output_file_path),
         _use_mvcc(use_mvcc),
-        _query_results_by_query_id() {}
+        _query_results_by_query_id(),
+        _enable_visualization(enable_visualization) {}
```

Comment:
personally, i would have called this „visualize plans“ but we don’t need to change this